### ISSUE TITLE
Using Aurora model for prediction for a specific location

### ISSUE THREAD

**Issue by MahanAbbassi** [microsoft/aurora#178]
I am working on adapting Aurora for station-level prediction, where the target corresponds to a single fixed geographic location (one longitude/latitude pair).

Since Aurora applies spatial patching in the encoder, I was wondering:
- Is it supported to run the model using inputs corresponding to only a single grid cell (i.e., one lon/lat pair)?
- Or does the patching mechanism impose a minimum spatial resolution / minimum grid size that must be provided as input?

More specifically, is there a required minimum spatial extent (e.g., H*W grid) for the encoder to function correctly, even if the prediction target is a single station?
Any guidance on the recommended setup for station-level or point-based prediction would be greatly appreciated.

**Comment by wesselb** [comment]
Hey @MahanAbbassi! These are very good questions.

In theory, the only constraint is that the input grid is a multiple of the patch size. Hence, if the patch size is `4`, then inputs of shape `(4, 4)` should be accepted.

In practice, there are a few other considerations:

1. If you're intending to use an existing pretrained or fine-tuned model, then these models will require global data as input. They will not work out-of-the-box if you give them data of limited spatial extent (like a local region).

2. If you're fine-tuning the model to work on a local region as input, you'll need to keep in mind that the model won't be able to see or model what's going on outside the region, so the predictions near the edge of the region will deterioriate as the lead time increases. The smaller the region, the more problematic this will be. (The solution here is to couple the local version of Aurora to a global model that provides the "boundary conditions", but this is a much more complicated setup.)

3. There _could_ be off-by-one errors which will not allow you to entirely go down to `(4, 4)`. We haven't tested these edge cases. Therefore, while it should work in principle, you might run into edge cases.

>  Any guidance on the recommended setup for station-level or point-based prediction would be greatly appreciated.

My advise would be a two-stage approach: run Aurora on global data, and then feed the global predictions into a downscaling model that outputs station-level forecasts. If the downscaling model is also a neural network, Aurora and the downscaling model could jointly be trained on station-level observations by propagating gradients all the way through.
